WordPress/Classes/ViewRelated/EEUUSCompliance/CompliancePopoverViewModel.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import UIKit
+import WordPressUI
+
+final class CompliancePopoverViewModel: ObservableObject {
+    @Published
+    var isAnalyticsEnabled: Bool = !WPAppAnalytics.userHasOptedOut()
+    var coordinator: CompliancePopoverCoordinator?
+
+    private let defaults: UserDefaults
+
+    init(defaults: UserDefaults) {
+        self.defaults = defaults
+    }
+
+    func didTapSettings() {
+        coordinator?.navigateToSettings()
+        defaults.shouldShowCompliancePopup = false
+    }
+
+    func didTapSave() {
+        let appAnalytics = WordPressAppDelegate.shared?.analytics
+        appAnalytics?.setUserHasOptedOut(!isAnalyticsEnabled)
+
+        let account = ContextManager.shared.performQuery { context -> WPAccount? in
+           let account = try? WPAccount.lookupDefaultWordPressComAccount(in: context)
+           return account
+        }
```

Comment:
Passing an `NSManagedObject` out of `performQuery` is likely to cause bugs – WDYT about passing out `account.userID` and `account.wordPressComRestApi` instead? (FWIW I suspect that `account.wordPressComRestApi` could be instantiated another way, rather than via the `NSManagedObject`)